### ISSUE TITLE
[FR] XEP-0277: Microblogging over XMPP

### ISSUE THREAD

**Issue by WPFilmmaker** [dino/dino#773]
It would be very nice being able to have such feature for leaving short messages (emergencies etc.)
reference: https://xmpp.org/extensions/xep-0277.html

**Comment by kousu** [comment]
Dino is a good chat client. It doesn't need to do everything XMPP can. Movim already exists for microblogging.

**Comment by WPFilmmaker** [comment]
Maybe I was not clear in explaining my request. I was not asking for a social network like movim, but rather the ability to have a function to write short messages (a-la twitter) visibile to the contacts. Dino's core function is messaging and I agree it should remain so, this would be an addition users would be free to use, or not to use, if needed :)

**Comment by licaon-kter** [comment]
@WPFilmmaker any mockups on how you want this to work/display?

**Comment by WPFilmmaker** [comment]
@licaon-kter right now there is a lot of space between the + button and the menu button, so maybe in that space?

https://i.imgur.com/CEp2ULo.png

A simple icon (e.g. a 'B' for blogging) would do. Then from there once you click you can see the available posts (if any) or write your own.

Maybe a small dot could appear if new posts are present. But no pop-ups or sounds, Dino is a messenger and should remain so, no bells and whistles :)

**Comment by licaon-kter** [comment]
Yes, but that's half the story, that's what you post only or what others in your roster posted or or or?
